JavaScriptCode.js

```diff
@@ -0,0 +1,368 @@
+var _ = require('lodash');
+const prompt = require('prompt-sync')();
```

Comment:
- We should always use import instead of require (if possible)
- For lodash try to import only the things we need,because lodash is a big module we need to import only the functions we are using

Reply:
noted

Reply:
done